.github/workflows/developer_onboarding_notification.yml

```diff
@@ -0,0 +1,70 @@
+name: PR Merge Comment
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions: write-all
+
+jobs:
+  comment-on-merge:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'develop'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set environment variables
+        env:
+          AUTHOR: ${{ github.event.pull_request.user.login }}
+          REPO: ${{ github.event.repository.name }}
+          OWNER: ${{ github.event.repository.owner.login }}
+        run: |
+          echo "AUTHOR=${AUTHOR}" >> $GITHUB_ENV
+          echo "REPO=${REPO}" >> $GITHUB_ENV
+          echo "OWNER=${OWNER}" >> $GITHUB_ENV
+
+      - name: Count merged PRs
+        id: count-prs
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const author = process.env.AUTHOR;
+            const repo = process.env.REPO;
+            const owner = process.env.OWNER;
+            const { data } = await github.rest.search.issuesAndPullRequests({
+              q: `repo:${owner}/${repo} type:pr state:closed author:${author}`
+            });
+            const prCount = data.items.filter(pr => pr.pull_request.merged_at).length;
+            core.exportVariable('PR_COUNT', prCount);
+
+      - name: Comment on the PR
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const prCount = parseInt(process.env.PR_COUNT);
+            const author = process.env.AUTHOR;
+            const mention = 'HardikGoyal2003';  // Update this if the mention needs to change
```

Comment:
Can we include more mentions to ensure it gets seen, if in case one of the mentions is not active ? WDYT ?

Reply:
We can add as many as you like. I initially included Afzal, but removed him due to unavailability. Once we have more onboarding leads or additional members in the welfare team, please let me know, and I will update it. Thanks!